nipap-cli/nipap

```diff
@@ -74,7 +74,12 @@ if __name__ == '__main__':
         sys.exit(1)
 
     # read configuration
-    cfg = ConfigParser.ConfigParser()
+    cfg_defaults = {
+        'scheme': 'http',
+        'hostname': 'localhost',
+        'port': '5000',
```

Comment:
Why 5000? nipapd binds to 1337 per default. It's paster for the web-ui that does 5000 per default, but nipap-cli doesn't talk to the web ui..